translations/translations.go

```diff
@@ -2,46 +2,46 @@ package translations
-	"処理中":             "Delivery Pending",
-	"au その他失敗":        "Error status for SDP",
-	"ドコモその他失敗":       "Error status for Docomo API",
-	"ドコモ圏外・OFF":      "Error status for Docomo API",
-	"ドコモ自 NW 障害":     "Error status for Docomo API",
-	"ドコモ他 NW 障害":     "Error status for Docomo API",
-	"無効":              "Invalid records",
-	"ソフトバンクその他失敗":    "Error status for Softbank API",
-	"ソフトバンク圏外・OFF":   "Error status for Softbank API",
-	"ソフトバンク自 NW 障害":  "Error status for Softbank API",
-	"ソフトバンク他 NW 障害":  "Error status for Softbank API",
-	"ソフトバンク受信拒否":     "Error status for Softbank API",
-	"送信不可":            "Error status for HLR API",
-	"送信前":             "SMS is scheduled, but not sent yet",
-	"一時停止":            "CSV file with SMS was paused",
-	"SMS 送信許可時間外":     "SMS was sent out of the allowed time",
-	"双方向 SMS 利用不可":    "No resourse for Reply",
-	"ドコモその他不明":       "Error status for Docomo API",
-	"重複不可":            "Duplicated sms",
-	"送信元重複":           "Keep chatID error",
-	"ドコモ一部受信":        "Error status for Docomo API",
-	"-":               "他人接続判定使用なし",
-	"キャンセル済":          "Cancelled SMS",
-	"送信上限":            "Ovelimitted SMS",
-	"NG ワード":         "Filtered SMS by stop words",
-	"配信停止希望":          "opted out",
-	"長文一部受信":          "Partial delivered via Gate",
-	"承認待ち":            "Wait for confirmation",
-	"Rakuten その他失敗":   "Error status for Rakuten",
-	"Rakuten 他 NW 障害": "Error status for Rakuten",
-	"Rakuten 一部受信":    "Error status for Rakuten",
-	"Rakuten 圏外・OFF":  "Error status for Rakuten",
+	"0":  "Failure",
+	"1":  "Sent",
+	"2":  "Sending",
+	"3":  "Redirect from shortened URL to target web sites",
+	"4":  "Not sent yet",
+	"5":  "Mobile numbers are in the Black list",
+	"7":  "Error status from SDP",
+	"8":  "Error status from SDP",
+	"9":  "Error status from SDP",
+	"10": "Error status from SDP",
+	"11": "Delivery Pending",
+	"12": "Error status for SDP",
+	"13": "Error status for Docomo API",
+	"14": "Error status for Docomo API",
+	"15": "Error status for Docomo API",
+	"16": "Error status for Docomo API",
+	"17": "Invalid records",
+	"18": "Error status for Softbank API",
+	"19": "Error status for Softbank API",
+	"20": "Error status for Softbank API",
+	"21": "Error status for Softbank API",
+	"22": "Error status for Softbank API",
+	"23": "Error status for HLR API",
+	"24": "SMS is scheduled, but not sent yet",
+	"25": "CSV file with SMS was paused",
+	"27": "SMS was sent out of the allowed time",
+	"28": "No resourse for Reply",
+	"29": "Error status for Docomo API",
+	"30": "Duplicated sms",
+	"31": "Keep chatID error",
+	"32": "Error status for Docomo API",
+	"33": "他人接続判定使用なし",
+	"34": "Cancelled SMS",
+	"35": "Ovelimitted SMS",
+	"36": "Filtered SMS by stop words",
+	"37": "opted out",
+	"38": "Partial delivered via Gate",
+	"39": "Wait for confirmation",
+	"40": "Error status for Rakuten",
+	"41": "Error status for Rakuten",
+	"42": "Error status for Rakuten",
+	"43": "Error status for Rakuten",
```

Comment:
For all these error messages that are duplicated, can we add the details from the original Japanese status?
You can ask Yuka to translate these.

For example:
`"43": "Rakuten out of reception zone"`